### PR TITLE
Code changes to use TOKEN_AUDIENCE_FORMAT

### PR DESCRIPTION
**Pull request by pgbhagat**
## Description

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](./CONTRIBUTING.md).**
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR.
- [x] The pull request does not introduce breaking changes (unless a major version change occurs in the assembly and module).
- [x] If applicable, the public code is properly documented.
- [x] Pull request includes test coverage for the included changes.
- [x] The code builds without any errors.

### CODE REVIEW

**Comment by pgbhagat** [comment]
MessageReceiver to use the same TOKEN_AUDIENCE_FORMAT as MessageSender

**Comment by SreeramGarlapati** [comment]
Hi @pgbhagat - can you please rebase (resolve conflicts).

File: azure-eventhubs/src/test/java/com/microsoft/azure/eventhubs/sendrecv/SendTest.java [hunk 1]
```diff
@@ -172,7 +172,7 @@ protected PartitionKeyValidator(final CompletableFuture<Void> validateSignal, fi
 		@Override
 		public void onReceive(Iterable<EventData> events)
 		{
-			if (events != null & events.iterator().hasNext())
+			if (events != null && events.iterator().hasNext())
```
Comment (SreeramGarlapati) [inline azure-eventhubs/src/test/java/com/microsoft/azure/eventhubs/sendrecv/SendTest.java]: nice find.

File: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/MessageSender.java [hunk 2]
```diff
@@ -828,7 +828,7 @@ private void throwSenderTimeout(final CompletableFuture<Void> pendingSendWork, f
 
         final boolean isClientSideTimeout = (cause == null || !(cause instanceof EventHubException));
         final EventHubException exception = isClientSideTimeout
-                ? new TimeoutException(String.format(Locale.US, "%s %s %s.", MessageSender.SEND_TIMED_OUT, " at ", ZonedDateTime.now(), cause))
```
Comment (SreeramGarlapati) [inline azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/MessageSender.java]: why - remove the cuase - exception?

File: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/amqp/BaseLinkHandler.java [hunk 3]
```diff
@@ -38,39 +37,35 @@ public void onLinkLocalClose(Event event) {
     public void onLinkRemoteClose(Event event) {
         final Link link = event.getLink();
 
-        if (link.getLocalState() != EndpointState.CLOSED) {
-            link.close();
-        }
-
         if (link != null) {
```
Comment (SreeramGarlapati) [inline azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/amqp/BaseLinkHandler.java]: In the onLinkXXX event handlers - link cannot be null. I introduced these null checks - when we had a bug - due to which reactor stack ran in multi-threaded fashion - and later missed to remove them. 
lets get rid of the if (link != null) code.

File: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/MessageSender.java (hunk 2 above)
Comment (pgbhagat) [reply azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/MessageSender.java]: Yes. i will add cause with one more formatter %s to String.format. Earlier no of argument to String.format and '%s' were not matching.

File: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/amqp/BaseLinkHandler.java (hunk 3 above)
Comment (pgbhagat) [reply azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/amqp/BaseLinkHandler.java]: Ok. let us remove null check. But having this in place is no harm.

**Comment by pgbhagat** [comment]
Raised a new pull request resolving the conflicts, i am closing this pull request

**Pull request closed without merging**